compiler/Runtime.cpp

```diff
@@ -163,7 +164,7 @@ bool isInterceptedFunction(const Function &f) {
       "malloc",   "calloc",  "mmap",    "mmap64", "open",   "read",    "lseek",
       "lseek64",  "fopen",   "fopen64", "fread",  "fseek",  "fseeko",  "rewind",
       "fseeko64", "getc",    "ungetc",  "memcpy", "memset", "strncpy", "strchr",
-      "memcmp",   "memmove", "ntohl",   "fgets",  "fgetc", "getchar"};
+      "memcmp",   "memmove", "ntohl",   "fgets",  "fgetc",  "getchar"};
```

Comment:
Chunk is unnecessary :)

Reply:
Not sure I understand; do you mean the change wasn't necessary? This is just `clang-format` doing its thing...